test/unit_tests_objects/active_time_ranges_test.dart

```diff
@@ -0,0 +1,30 @@
+/// Copyright 2020 Google LLC
```

Comment:
Should be test/unit_tests/objects/active_time_ranges_test.dart 
The objects should be in sub folder of the unit_tests

Reply:
Done.